lib/parsers/asm-parser-spirv.ts

```diff
@@ -98,7 +98,7 @@ export class SPIRVAsmParser extends AsmParser {
         const endTime = process.hrtime.bigint();
         return {
             asm: asm,
-            labelDefinitions: {},
+            labelDefinitions: new Map<string, number>(),
```

Comment:
ditto this - is this something separable?

Reply:
Probably